suncityla/app/layout.tsx

```diff
@@ -31,7 +31,7 @@ export default async function RootLayout({
   return (
     <html lang="en">
       <body className={`${geistSans.variable} ${geistMono.variable} antialiased`}>
-        <div className="container mx-auto ">
```

Comment:
If we don't have a container the pages won't work well I. Large screens. Are you handling this somewhere else instead?

Reply:
I removed this because it created a margin/padding on the whole page. I have made the landing page responsive now. You can check it out.

Reply:
I think we can add it like this: lg:container since the issue resides with large screens?